src/vm/finalizerthread.cpp

```diff
@@ -1284,16 +1284,6 @@ BOOL FinalizerThread::FinalizerThreadWatchDog()
         
         BOOL fTimeOut = (status == WAIT_TIMEOUT) ? TRUE : FALSE;
 
-#ifndef GOLDEN
-        if (fTimeOut) 
-        {
-            if (dwBreakOnFinalizeTimeOut) {
```

Comment:
Either remove everything related to `dwBreakOnFinalizeTimeOut`, or just remove `ifndef GOLDEN`. I would remove just the `ifndef GOLDEN`.